src/server/api/routers/clubEdit.ts

```diff
@@ -0,0 +1,177 @@
+import { db } from '@src/server/db';
+import { createTRPCRouter, protectedProcedure } from '../trpc';
+import { and, eq, inArray, sql } from 'drizzle-orm';
+import { club, contacts, userMetadataToClubs } from '@src/server/db/schema';
+import { editClubSchema } from '@src/utils/formSchemas';
+import { TRPCError } from '@trpc/server';
+import { z } from 'zod';
+import { selectContact } from '@src/server/db/models';
+async function isOfficer(userId: string, clubId: string) {
+  const officer = await db.query.userMetadataToClubs.findFirst({
+    where: and(
+      eq(userMetadataToClubs.userId, userId),
+      eq(userMetadataToClubs.clubId, clubId),
+    ),
+  });
+  return officer?.memberType == 'Officer' || 'President';
+}
```

Comment:
This currently returns `true` or `'President'`. Rework the if statement so it properly works